packages/ember-application/lib/system/application.js

```diff
@@ -1097,4 +1031,36 @@ function resolverFor(namespace) {
   return resolve;
 }
 
+function registerLibraries() {
+  if (!librariesRegistered) {
+    librariesRegistered = true;
+
+    if (environment.hasDOM) {
+      Ember.libraries.registerCoreLibrary('jQuery', jQuery().jquery);
+    }
+  }
+}
+
+function logLibraryVersions() {
```

Comment:
@fivetanley - I agree in principle, but having `Ember.debug` nested inside of `Ember.runInDebug` would throw errors (because defeatureify doesn't like nested stripped blocks).  Some more finagling would be required to deal with that (still totally doable though).